akka-sample-javaslang/src/main/java/sample/javaslang/Master.java

```diff
@@ -0,0 +1,54 @@
+package sample.javaslang;
+
+import akka.actor.Props;
+import akka.actor.UntypedAbstractActor;
+import javaslang.collection.List;
+import javaslang.collection.Stream;
+
+import static sample.javaslang.Minion_StoppedPatterns.*;
+
+import static javaslang.API.*;
+import static javaslang.Predicates.*;
+
+public class Master extends UntypedAbstractActor {
+
+  private final Maze maze;
+
+  private List<Minion.Stopped> results = List.empty();
+
+  public Master(Maze maze) {
+    this.maze = maze;
+  }
+
+  public void onReceive(Object message) {
+    Match(message).of(
+      Case(instanceOf(Start.class), s -> {
+        Stream.range(0, s.count).forEach(c ->
+          getContext().actorOf(Props.create(() -> new Minion(this.maze)))
+        );
+        return null;
```

Comment:
[Supposedly](http://www.javaslang.io/javaslang-docs/#_the_basics_of_match_for_java) there's a [`run()`](https://github.com/javaslang/javaslang/blob/master/javaslang/src-gen/main/java/javaslang/API.java#L2281) static method that allows you to turn a `Runnable` into something returning `Void`, so you don't have to write `return null`.